oshell.sh

```diff
@@ -144,6 +201,44 @@ function oci_env_print() {
   return 0
 }
 
+# Display OCI tenancy map with formatted colors
+alias ocimap='oci_tenancy_map'
+function oci_tenancy_map() {
+  local yaml_file="${1:-$HOME/.oci/tenancy-map.yaml}"
```

Comment:
This function appears to break when called from `oci_authenticate`:
<img width="384" alt="image" src="https://github.com/user-attachments/assets/9ec0d058-11a3-4441-8152-59a9d7047d37" />

And not sure we would want to handle a non-default location.

```suggestion
  local tenancy_map_path="${HOME}/.oci/tenancy-map.yaml"
```